### DISCUSSION TITLE
Icon request: hourglass

### DISCUSSION THREAD

**Discussion by borisdayma** [tailwindlabs/heroicons#408] [Ideas]
Those icons are great!
It would be nice if you had an hourglass for things such as "loading" or "in progress"

**Comment by vhscom** [comment]
Considering the [`animate-spin` option in Tailiwind CSS](https://tailwindcss.com/docs/animation#spin) what I'd prefer to see is a static loading icon we can _spin_ to show progress as requested here: https://github.com/tailwindlabs/heroicons/discussions/603. Hourglasses make me think of Windows 3.1.

**Reply by mdwheele** [reply]
Another use-case for an hourglass icon is a "waiting" state. I have an example where several task are waiting for another task to complete and my goto is to use a "pause" icon, but then you send the message that the user can take action by pausing; not quite there.

An hourglass sends the message that "We're just chilling..."
